src/utils/parse_utils.ts

```diff
@@ -55,7 +55,8 @@ export const parseUtils = {
             return {
                 excludedSources: parseUtils.parseStringArrForERC20BridgeSources(excludedIds),
                 includedSources: [],
-                nativeExclusivelyRFQT: false,
+                // Exclude open orderbook if 'Mesh' is excluded.
+                nativeExclusivelyRFQT: excludedIds.includes('Mesh'),
```

Comment:
But only if they pass in `excludedSources=Native,Mesh`, right? I think `excludedSources=Mesh` should just turn into
```js
{
  excludedSources: [],
  includedSources: [],
  nativeExclusivelyRFQT: true
}
```

Reply:
Yup that is correct. Do you think it's worth changing to `excludedIds.includes('Mesh') && !excludedIds.includes('Native')`?

Reply:
not worth including in this PR, but we should consider just having `Mesh` and `RFQt` as separate enum values in `ERC20BridgeSource`

Reply:
> Yup that is correct. Do you think it's worth changing to `excludedIds.includes('Mesh') && !excludedIds.includes('Native')`?

Doesn't it make more sense to let that case error out rather than silently succeed?

Reply:
good point, as long as it bubbles up as an API error and fails gracefully, I'm OK with that